website/docs/r/policy_edge_transport_node.html.markdown

```diff
@@ -84,6 +111,7 @@ The following arguments are supported:
 * `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
 * `site_path` - (Optional) The path of the site which the Edge Transport Node belongs to. `path` field of the existing `nsxt_policy_site` can be used here. Defaults to default site path.
 * `enforcement_point` - (Optional) The ID of enforcement point under given `site_path` to manage the Edge Transport Node. Defaults to default enforcement point.
+* `node_id` - (Optional) The id of a pre-deployed Edge appliance to be converted into a Policy Edge transport node. Note that `node_id` attribute conflicts with `advanced_configuration`, `appliance_config`, `credentials`, `form_factor`, `management_interface`, and `vm_deployment_config`. These attributes will be ignored while specifying `node_id`.
```

Comment:
I would remove the last sentence, since terraform validation will fail due to `ConflictsWith`

Reply:
Indeed.